pkg/controller/launch/controller.go

```diff
@@ -92,6 +92,18 @@ func (lr *LaunchReconciler) reconcilePool(kluster *v1.Kluster, pool *models.Node
 		requeue = true
 	case status.Stopping > 0:
 		requeue = true
+	case status.Errored > 0:
+		for i := 0; i < status.Errored; i++ {
+			requeue = true
+			errNode := status.Nodes[i]
+			if err = pm.ResetNodeState(errNode); err != nil {
+				return
```

Comment:
Well.. Since you created a ticket for this, I though there must be more to it and I read in various places that VMs in `error` cannot be deleted and have to be reset beforehand. But I haven't tested this yet.